apps/explorer/src/styles.css

```diff
@@ -1,4 +1,26 @@
 /* You can add global styles to this file, and also import other style files */
 @tailwind base;
 @tailwind components;
+
+.route-header {
```

Comment:
As per other comment, lets embrace tailwind a bit more. Lets just add components for these styles.